tests/vxlan/test_vxlan_ecmp.py

```diff
@@ -1537,6 +1557,27 @@ def test_vxlan_remove_add_underlay_default(self,
            tc14: add the underlay default route.
         '''
         self.setup = setUp
+        duthost = self.setup['duthost']
+        vnet = self.setup[encap_type]['vnet_vni_map'].keys()[0]
+
+        # setup route with 3 VNET endpoints
+        n_endpoints = 3
+        endpoint_group_C = [
+            ecmp_utils.get_ip_address(af=ecmp_utils.get_outer_layer_version(encap_type), netid=NEXTHOP_PREFIX)
+            for _ in range(n_endpoints)
+        ]
+        route_prefix = ecmp_utils.get_ip_address(
+            af=ecmp_utils.get_payload_version(encap_type), netid=DESTINATION_PREFIX
+        )
+        ecmp_utils.set_routes_in_dut(
+            duthost,
+            {vnet: {route_prefix: endpoint_group_C}},
+            ecmp_utils.get_payload_version(encap_type),
+            'SET',
+            bfd=self.setup['enable_bfd']
+        )
+        self.setup[encap_type]['dest_to_nh_map'][vnet][route_prefix] = endpoint_group_C
+
```

Comment:
@antonptashnik These two parts of code looks same, could you please add a common function for it?